EIPS/eip-1776.md

```diff
@@ -0,0 +1,73 @@
+---
+eip: 1776
```

Comment:
```suggestion
eip: 4713
```
EIP numbers are assigned by editors, not selected by authors.  We do not allow choosing numbers.